Formula/kibana-full.rb

```diff
@@ -23,7 +23,13 @@ def install
       "x-pack",
     )
 
-    cd prefix do
+    Pathname.glob(libexec/"bin/*") do |f|
```

Comment:
Do you know if this is something we can do on line 11 too? https://github.com/elastic/homebrew-tap/issues/6